src/spec/example.cr

```diff
@@ -47,6 +47,8 @@ module Spec
     rescue ex : Spec::AssertionFailed
       @parent.report(:fail, description, file, line, Time.monotonic - start, ex)
       Spec.abort! if Spec.fail_fast?
+    rescue ex : Spec::ExamplePending
+      @parent.report(:pending, description, file, line, Time.monotonic - start)
```

Comment:
`ex.message` is not being used anywhere, is that intentional?

Reply:
For now yes.

While I felt it important to be able to specify a reason in the `pending!` call, making it clearer why that call is happening all of the sudden, we currently have no holistic approach to pending reasons. That is the `pending` method does not accept a reason in any form and there's no existing reporting format/infrastructure for this. I think `pending`  should gain a way to collect reasons and then we should update the formatter to present them, but I felt introducing that holistic approach here to be a bit too much for what it wants to do, also because there's probably a lot more opinion on how that should be done.